### ISSUE TITLE
Make sure multiple elements (without nesting) are on one line

### ISSUE THREAD

**Issue by spenno** [brigade/scss-lint#456]
At the moment, the linter doesn't seem to mind if multiple elements span over multiple lines (when not nested). E.g:

```sass
.active

span {
    background: transparent;
}
```

I wonder if this should be caught by the linter?

**Comment by sds** [comment]
Hey @spenno,

This sounds like a good idea to me since oddly structured selectors like that can make it difficult to read, especially when combined with multiple comma-separated sequences. Would happily accept a pull request adding a linter for this, otherwise we'll get to it when we can.

**Comment by sds** [comment]
Extended the `SingleLinePerSelector` linter to enforce sequences occupy a single line in f99c7d064fa98b1200bf6eab6235a8f89c1d10ef.
